rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -259,3 +272,16 @@ func (r *Layer1Relayer) shouldUpdateGasOracle(baseFee uint64, blobBaseFee uint64
 
 	return false
 }
+
+func (r *Layer1Relayer) commitBatchReachTimeout() bool {
+	fields := map[string]interface{}{
+		"rollup_status": types.RollupCommitted,
+	}
+	orderByList := []string{"updated_at DESC"}
+	limit := 1
+	batches, err := r.batchOrm.GetBatches(r.ctx, fields, orderByList, limit)
```

Comment:
the SQL may be slow, suggesting queries like `order by index desc limit 1`, utilizing the primary key, the batch with latest index in status set (committed, finalizing, finalized) surely has the largest committed time, because batches are to be committed one by one.